examples/wallet/.eslintrc.json

```diff
@@ -7,8 +7,12 @@
       }
     }
   },
+  "env": {
+    "jest": true
+  },
   "rules": {
     "react/require-default-props": "off",
+    "func-names": "off",
```

Comment:
what's this for?

Reply:
to have the linter accept anonymous non-arrow functions. It is usually the case with the tests, more so with mocha.

We don't use mocha though. should I delete it?